### DISCUSSION TITLE
Failed to resolve

### DISCUSSION THREAD

**Discussion by lkzwc** [web-infra-dev/rspress#196] [Q&A]
![image](https://github.com/web-infra-dev/rspress/assets/84896877/02d9342d-e177-4f36-bb22-df17d1e3110b)

**Comment by sanyuan0704** [comment]
@components 需要设置一下路径别名

**Reply by lkzwc** [reply]
改成绝对地址  解析报错了  需要安装其他的吗
![image](https://github.com/web-infra-dev/rspress/assets/84896877/c3db7456-ceb1-4cbb-8b98-b6b58c660628)

**Comment by lkzwc** [comment]
![image](https://github.com/web-infra-dev/rspress/assets/84896877/59457970-954a-476e-89eb-3a138605009b)
